app/Keycloak/Service/IntegrationUrlConverter.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Keycloak\Service;
+
+use App\Domain\Integrations\Integration;
+use App\Domain\Integrations\IntegrationPartnerStatus;
+use App\Domain\Integrations\IntegrationUrl;
+use App\Domain\Integrations\IntegrationUrlType;
+use App\Keycloak\Client;
+
+/* Converts integration urls in the correct Keycloak API format */
+final readonly class IntegrationUrlConverter
+{
+    public static function convert(Integration $integration, Client $client): array
+    {
+        // Empty start construct to make all urls empty at the beginning of each update
+        $urls = [
+            'baseUrl' => '',
+            'redirectUris' => [],
+            'attributes' => ['post.logout.redirect.uris' => ''],
+            'webOrigins' => [
+                '+', // This permits all origins of Valid Redirect URIs for CORS checks
+            ],
+        ];
+
+        if ($integration->partnerStatus !== IntegrationPartnerStatus::FIRST_PARTY) {
```

Comment:
I wonder if we should do the same for Auth0 and UiTiD?

Reply:
Discussed it briefly with Jonas here. UiTiDv1 does not have the concept of 1st vs 3th party clients, so no issues there according to Jonas. In auth0 it would be a nice check, but it does not give issues currently and I would not put time in it because it will be replaced in less then a year anyway.